entities/attribute.py

```diff
@@ -0,0 +1,139 @@
+from typing import Optional
+
+from entities.common import EnumValue, Value
+
+
+class AttributeUnit(EnumValue):
+    def get_name(self) -> str:
+        return 'unit'
+
+
+class UnitlessUnit(AttributeUnit):
+    NONE = "none"
+
+
+class AttributeType(EnumValue):
+    VOLTAGE = 'voltage'
+    STRING = 'string'
+    CURRENT = 'current'
+    RESISTANCE = 'resistance'
+    CAPACITANCE = 'capacitance'
+    POWER = 'power'
+    INDUCTANCE = 'inductance'
+    FREQUENCY = 'frequency'
+
+    def get_name(self) -> str:
+        return 'type'
+
+
+class Attribute():
+    def __init__(self, name: str, value: Value | str, attribute_type: AttributeType, unit: Optional[AttributeUnit]) -> None:
```

Comment:
```suggestion
    def __init__(self, name: str, value: Union[Value, str], attribute_type: AttributeType, unit: Optional[AttributeUnit]) -> None:
```

The `|` operator for types isn't supported in Python 3.8 which we do still support.

Reply:
Good catch! 👍

Reply:
Also it needs to be imported first from `typing`.

Reply:
Yup, sorry! 😅 Oversight on my side